gcsweb/cmd/gcsweb/gcsweb.go

```diff
@@ -371,7 +373,7 @@ func (s *server) gcsRequest(w http.ResponseWriter, r *http.Request) {
 
 	// This means that the object is a file.
 	if objAttrs != nil {
-		if err := s.handleObject(w, bucket, object); err != nil {
+		if err := s.handleObject(w, bucket, object, objAttrs.ContentType); err != nil {
```

Comment:
Probably want to pass through `ContentEncoding` too?

Reply:
I include also the encoding.